src/ensembl/src/content/app/browser/browserActions.ts

```diff
@@ -5,62 +5,124 @@ import { TrackType } from './track-panel/trackPanelConfig';
 export const updateBrowserActivated = createAction(
   'browser/update-browser-activated',
   (resolve) => {
-    return (browserActivated: boolean) => resolve(browserActivated);
+    return (browserActivated: boolean) =>
+      resolve(browserActivated, {
+        ga: {
+          category: 'Browser',
+          label: 'Default Action'
+        }
+      });
   }
 );
 
 export const toggleTrackPanel = createAction(
   'browser/toggle-track-panel',
   (resolve) => {
-    return (trackPanelOpened?: boolean) => resolve(trackPanelOpened);
+    return (trackPanelOpened?: boolean) =>
+      resolve(trackPanelOpened, {
+        disable_tracking: true,
+        ga: {
+          category: 'Track Panel',
+          label: 'User Interaction'
+        }
+      });
   }
 );
 
 export const changeDrawerView = createAction(
   'browser/change-drawer-view',
   (resolve) => {
-    return (drawerView: string) => resolve(drawerView);
+    return (drawerView: string) =>
+      resolve(drawerView, {
+        ga: {
+          category: 'Drawer',
+          label: 'User Interaction'
+        }
+      });
   }
 );
 
 export const toggleDrawer = createAction('browser/toggle-drawer', (resolve) => {
-  return (drawerOpened?: boolean) => resolve(drawerOpened);
+  return (drawerOpened?: boolean) =>
+    resolve(drawerOpened, {
+      ga: {
+        category1: 'Drawer',
```

Comment:
is `category1` intentional or is it a typo?

Reply:
Well spotted! It was intentional to test the warning message but I missed to revert it. I've reverted it now.